src/Common/MoistThermodynamics/states.jl

```diff
@@ -185,8 +205,10 @@ function LiquidIcePotTempSHumEquil_given_pressure(
     p::FT,
     q_tot::FT,
     maxiter::Int = 30,
-    tol::FT = FT(1e-1),
+    temperature_tol::FT = FT(1e-3),
```

Comment:
```suggestion
    potential_temperature_tol::FT = FT(1e-2),
```
(Please do not make this too small. We do not have temperatures or potential temperatures to greater accuracy than 1e-2 K.)